### PR TITLE
Website: Update properly to Kubernetes 1.16

### PR DESCRIPTION
**Pull request by aLekSer**
Add missing fleet.yaml update. Properly update website md files.

**What type of PR is this?**

/kind documentation

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:
Work on #1649.

**Special notes for your reviewer**:
Sorry for leaving this in a previous PR.

### CODE REVIEW

File: site/content/en/docs/Reference/fleet.md [hunk 1]
```diff
@@ -12,6 +12,66 @@ Like any other Kubernetes resource you describe a `Fleet`'s desired state via a
...
+        httpPort: 9358
+      # The GameServer's Pod template
+      template:
+        spec:
+          containers:
+          - name: simple-udp
+            image: gcr.io/agones-images/udp-server:0.21
+```
+{{% /feature %}}
+{{% feature publishVersion="1.9.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/fleet.md:74]: I don't see that the link was changed to 1.16 in the "new" block.

File: site/content/en/docs/Reference/fleetautoscaler.md [hunk 2]
```diff
@@ -9,6 +9,38 @@ weight: 30
...
+      bufferSize: 5
+      # minimum fleet size to be set by this FleetAutoscaler. 
+      # if not specified, the actual minimum fleet size will be bufferSize
+      minReplicas: 10
+      # maximum fleet size that can be set by this FleetAutoscaler
+      # required
+      maxReplicas: 20
+```
+{{% /feature %}}
+{{% feature publishVersion="1.9.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/fleetautoscaler.md:43]: Same for this one.

File: site/content/en/docs/Reference/gameserver.md [hunk 3]
```diff
@@ -9,6 +9,88 @@ description: >
...
+        myspeciallabel: myspecialvalue
+    # Pod Specification
+    spec:
+      containers:
+      - name: simple-udp
+        image:  gcr.io/agones-images/udp-server:0.21
+        imagePullPolicy: Always
+```
+{{% /feature %}}
+{{% feature publishVersion="1.9.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Reference/gameserver.md:93]: here too.

File: site/content/en/docs/Reference/fleet.md (hunk 1 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md:74]: Wait a bit, this new block is a leftover of https://github.com/googleforgames/agones/pull/1787
So `expiryVersion` does contain 1.15 and `publishVersion` 1.16.

**Comment by aLekSer** [comment]
Please note that a previous [PR #1787 ](https://github.com/googleforgames/agones/pull/1787) contains an update to 1.16, and I forgot to use Hugo shortcode in it. That's why I added `expiryVersion` with 1.15.

File: site/content/en/docs/Reference/fleet.md (hunk 1 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/fleet.md:74]: The new block is what I missed from original PR #1787. So technically this is an old block so with `expiryVersion` .

File: site/content/en/docs/Reference/fleetautoscaler.md (hunk 2 above)
Comment (aLekSer) [reply site/content/en/docs/Reference/fleetautoscaler.md:43]: This is correct, please view the lines down this change they contain 1.16 change.

File: site/content/en/docs/Reference/fleet.md (hunk 1 above)
Comment (roberthbailey) [reply site/content/en/docs/Reference/fleet.md:74]: Makes sense. I didn't see that we had updated the live site to the 1.16 links.

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**